cmd/rook/filesystem/mount_test.go

```diff
@@ -38,7 +38,7 @@ func TestMountFilesystem(t *testing.T) {
 	}
 	e := &exectest.MockExecutor{
 		MockExecuteCommand: func(actionName string, command string, arg ...string) error {
-			expectedArgs := []string{"mount", "-t", "ceph", "-o", "name=admin,secret=AQBsCv1X5oD9GhAARHVU9N+kFRWDjyLA1dqzIg==",
```

Comment:
yes, the command is now `mount`.  Before, we had `sudo` baked in as the command, which is not correct and is also not present in the Alpine image.  If the caller doesn't have the privileges then mount should fail, `sudo` should not be baked into the call.